specification/source/glossary.rst

```diff
@@ -70,3 +85,13 @@ the Open Building Controls project.
    Mode
      In CDL, by mode we mean a signal that can take on multiple distinct
      values, such as ``On``, ``Off``, ``PreCool``.
+
+   Non-HVAC System
+     Any non-HVAC plant coupled with the control system.
+
+   Plant
```

Comment:
@milicag Is this "Plant" interpretation aligning with the one shown in the ISO standard, or any other source?